packages/core/src/textures/resources/SVGResource.js

```diff
@@ -40,6 +40,13 @@ export default class SVGResource extends BaseImageResource
          */
         this._resolve = null;
 
+        /**
+         * Cross origin value to use
+         * @private
+         * @member {boolean|string}
+         */
+        this._crossorigin = options.crossorigin || true;
```

Comment:
This seems like a problem, `crossorigin` will always be true here even if someone sets `false`. I would just leave this as `undefined` and let `BaseImageResource.crossOrigin` handle it.
```suggestion
        this._crossorigin = options.crossorigin;
```